materialize-sql/driver.go

```diff
@@ -318,6 +318,8 @@ func (d *Driver) NewTransactor(ctx context.Context, open pm.Request_Open, be *bo
 		}
 	}
 
+	log.WithField("eventType", "connectorStatus").Info("Started")
```

Comment:
I think we're going to want this in the [boilerplate file](https://github.com/estuary/connectors/blob/55b604c4d95f7b6baa6cd02ded414a131ae497ab/materialize-boilerplate/boilerplate.go#L165) so that it applies for all materializations, not just the ones using `materialize-sql`.